packages/clay-button/src/ButtonWithIcon.tsx

```diff
@@ -8,27 +8,14 @@ import React from 'react';
 
 import ClayButton from './Button';
 
-type ButtonAria =
-	| {
-			/**
-			 * Define a value that labels the button.
-			 */
-			'aria-label': string;
-			'aria-labelledby'?: never;
-	  }
-	| {
-			/**
-			 * Define a value that labels the button.
-			 */
-			'aria-label'?: never;
-			'aria-labelledby': string;
-	  };
-
 interface ICommonProps
 	extends Omit<
 		React.ComponentProps<typeof ClayButton>,
 		'aria-label' | 'aria-labelledby'
 	> {
+	'aria-label'?: string;
+
+	'aria-labelledby'?: string;
```

Comment:
The old type looked like it was requiring either aria-labelledby or aria-label to be included while also not allowing both to be added at the same time. This doesn't look like this will have the same behavior.

Reply:
For some reason with the updated type libraries, it no longer allowed them to be mutually exclusive. I'm not exactly why or what type changed in the react definitions, but I figured it might just be simpler to make them both optional types instead.